instrumentation/mongo/Appraisals

```diff
@@ -5,5 +5,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 appraise 'mongo-2.13' do
```

Comment:
Some of the mongo tests are failing on 3.4 with this update.

Seeing that the latest version is 2.21.0, what do you think about adding a new appraisal for the most recent version? 

Many instrumentation test suites have an appraisal without a version specified to make sure it's compatible with the latest release.

Reply:
Followed the pattern in other appraisal files (just doing major version when constrained to minor version). Pushed a commit to fix the tests, still digging in to impacts of those changes to generate a good changelog.

Reply:
Thanks!